static/css/mozsearch.css

```diff
@@ -667,6 +661,7 @@ span[data-symbols].hovered {
     -webkit-user-select: text;
     -ms-user-select: text;
     user-select: text;
+    /* transition: transform .1s ease; */
```

Comment:
remove if not needed?

Reply:
Yup, this was leftover, I wasn't sure if it'd look good, it doesn't :)